master/buildbot/status/master.py

```diff
@@ -307,7 +307,7 @@ def refill():
             candidates = [(i, b, b.getTimes()[1])
                           for i, b in enumerate(next_build)
                           if b is not None]
-            candidates.sort(lambda x, y: cmp(x[2], y[2]))
+            candidates.sort(key=lambda x: x[2])
```

Comment:
I agree with @rodrigc with use lambda is lot of place where it makes the code easier to read like here.